src/main/resources/assets/js/components/TaskListItemComponent.jsx

```diff
@@ -117,9 +134,13 @@ define([
             </span>
           </td>
           <td className="text-right">
-            <time dateTime={task.get("version").toISOString()} title={task.get("version").toISOString()}>
-              {task.get("version").toLocaleString()}
-            </time>
+            <span
+              title={task.get("version").toISOString()}
+              className={versionClassSet}
+              onClick={this.handleVersionClick}>{
+                this.state.versionShowTimestamp ?
```

Comment:
Yes it is :)

On Thu, Jan 8, 2015 at 8:22 AM, Felix Gertz <notifications@github.com>
wrote:

> In src/main/resources/assets/js/components/TaskListItemComponent.jsx
> <https://github.com/mesosphere/marathon/pull/952#discussion-diff-22661576>
> :
>
> > @@ -117,9 +134,13 @@ define([
> >              </span>
> >            </td>
> >            <td className="text-right">
> > -            <time dateTime={task.get("version").toISOString()} title={task.get("version").toISOString()}>
> > -              {task.get("version").toLocaleString()}
> > -            </time>
> > +            <span
> > +              title={task.get("version").toISOString()}
> > +              className={versionClassSet}
> > +              onClick={this.handleVersionClick}>{
> > +                this.state.versionShowTimestamp ?
>
> I've talked with @drexin <https://github.com/drexin> and removed the
> click interaction. It's enough to see the older versions in the
> configuration.
> So the 'variable above' is obsolete, isn't it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/marathon/pull/952/files#r22661576>.
>